bellows/ezsp/v8/config.py

```diff
@@ -22,7 +22,7 @@
     # The maximum number of router neighbors the stack can keep track of. A neighbor
     # is a node within radio range
     vol.Optional(EzspConfigId.CONFIG_NEIGHBOR_TABLE_SIZE.name): vol.All(
```

Comment:
should this get a default added for 16?

Reply:
Nope. Without setting a default it would use the value the NCP image was compiled with. 
And in other words I suspect that for v8 version and newer maybe it is better not to set any values at all? E.g. if the ncp image was compily with 16 tc keys slots, you won't be able to set a higher value. Essentially you only may decrease values, but not increase them. But I'm not positive this applies to all values.